packages/desktop-client/src/components/mobile/transactions/TransactionList.jsx

```diff
@@ -119,6 +145,244 @@ export function TransactionList({
           );
         })}
       </ListBox>
+      {selectedTransactions?.length > 0 && (
+        <FloatingActionBar
+          transactions={transactions}
+          selectedTransactions={selectedTransactions}
+          onClearSelectedTransactions={onClearSelectedTransactions}
+          onEdit={onBatchEdit}
+          onDuplicate={onBatchDuplicate}
+          onLinkSchedule={onLinkSchedule}
+          onUnlinkSchedule={onUnlinkSchedule}
+          onSetTransfer={onSetTransfer}
+          onDelete={onBatchDelete}
+        />
+      )}
     </>
   );
 }
+
+function FloatingActionBar({
+  transactions,
+  selectedTransactions,
+  onClearSelectedTransactions,
+  onEdit,
+  onDuplicate,
+  onLinkSchedule,
+  onUnlinkSchedule,
+  onDelete,
+  style,
+}) {
+  const editMenuTriggerRef = useRef(null);
+  const [isEditMenuOpen, setIsEditMenuOpen] = useState(false);
+  const moreOptionsMenuTriggerRef = useRef(null);
+  const [isMoreOptionsMenuOpen, setIsMoreOptionsMenuOpen] = useState(false);
+  const getMenuItemStyle = item => ({
+    ...styles.mobileMenuItem,
+    color: theme.mobileHeaderText,
+    ...(item.name === 'delete' && { color: theme.errorTextMenu }),
+  });
+
+  const buttonProps = {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏by moving this constant definition outside the component you _might_ reduce the amonut of renders due to the object being re-created each time.